runtime/drive_handler.go

```diff
@@ -378,3 +383,10 @@ func (sd stubDrive) PatchAndMount(
 
 	return nil
 }
+
+// CacheType sets the stub drive's cacheType value from the provided DriveMount configs.
+func (sd *stubDrive) WithCacheType(cacheType string) {
```

Comment:
setCacheType?

- WithXxx is mainly used for functional options.
- Seems we don't have to expose this method.